requirements.txt

```diff
@@ -1,7 +1,96 @@
+aioredis==1.3.1
+amqp==2.5.2
+appdirs==1.4.3
 asgiref==3.2.10
+async-timeout==3.0.1
+attrs==19.3.0
+autobahn==20.4.3
+Automat==20.2.0
+beautifulsoup4==4.8.2
+billiard==3.6.3.0
+braintree==4.1.0
+bs4==0.0.1
+cachetools==4.1.1
+celery==4.4.2
+certifi==2019.11.28
+cffi==1.14.0
+channels==2.4.0
+channels-redis==2.4.2
+chardet==3.0.4
+client==0.0.1
+constantly==15.1.0
+coreapi==2.3.3
+coreschema==0.0.4
+cryptography==2.9.2
+daphne==2.5.0
+defusedxml==0.6.0
+distlib==0.3.0
+dj-database-url==0.5.0
 Django==3.0.8
+django-allauth==0.41.0
+django-cart==1.0.4
+django-carton==1.2.1
+django-celery-results==1.2.1
+django-cors-headers==3.3.0
+django-crispy==1.8.1
+django-crispy-forms==1.9.0
+django-dotenv==1.4.2
+django-filter==2.2.0
+django-rest-auth==0.9.5
+django-rest-swagger==2.0.7
 djangorestframework==3.11.0
-Pillow==7.2.0
```

Comment:
Why did you remove pillow package?

Reply:
not intentional ,, that was when i updated my requirement.txt